packages/SwingSet/src/kernel/kernel.js

```diff
@@ -322,7 +325,7 @@ export default function buildKernel(kernelEndowments, kernelOptions = {}) {
     const { type } = parseKernelSlot(target);
     if (type === 'object') {
       const vatID = kernelKeeper.ownerOfKernelObject(target);
-      insistVatID(vatID);
+      vatID === 'none' || insistVatID(vatID);
```

Comment:
`insistVatID` was changed to tolerate `none`, is this bypass also necessary?

(I'm on the fence as to whether we add the null type to the definition of a VatID, which has one set of problems, or effectively create a new "vatID or None" type like this suggests)

Reply:
Yes, I think that check was a byproduct of churn during development.